eap-xp4-jdk17/runtime-image/test/Dockerfile

```diff
@@ -0,0 +1,7 @@
+#Be sure to update eap-testapp with the name of the image generated with s2i.
+FROM jboss-eap-7/eap-xp4-openjdk11-runtime-openshift-rhel8
```

Comment:
Should be the 17 image.

Reply:
@rdnovell , that is still present.

Reply:
Done in [commit](https://github.com/jboss-container-images/jboss-eap-7-openshift-image/pull/506/commits/9a2e8affaef64ab074872db79794d7c8279cba66)